spamostack/keeper.py

```diff
@@ -119,6 +120,10 @@ def get(self, client_name, resource_name, param=None, func=None,
                     else:
                         probe = getattr(el, param)(*args, **kwargs)
 
+                    if (resource_name == "flavors"
```

Comment:
All the resources have already a distinguish as a placement to the cache thus they could be cleaned up with `spamostack --clean all` for example. But flavors are an exception in sense the default flavors placed to cache for initial usage. So there is no need to use that prefix for all the resources or is there?